pkg/windows/windows.go

```diff
@@ -82,6 +88,8 @@ const (
 	cmdExitNoStatus = "command exited without exit status or exit signal"
 	// removeHNSCommand is the Windows command used to remove HNS network.
 	removeHNSCommand = "Remove-HnsNetwork"
+	//containerdRuntime is containerd runtime name
```

Comment:
Won't this also be the name of the service?

Reply:
Yes. This will be the service name